inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp

```diff
@@ -35,26 +35,63 @@ static inline size_t parallel_init(size_t start, size_t nDims, const SizeVector&
     return start;
 }
 
-MKLDNNStridedSliceNode::MKLDNNStridedSliceNode(const InferenceEngine::CNNLayerPtr& layer, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache) :
-        MKLDNNNode(layer, eng, cache) {}
+bool MKLDNNStridedSliceNode::isSupportedOperation(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        const auto ss = std::dynamic_pointer_cast<const ngraph::opset1::StridedSlice>(op);
+        if (!ss) {
+            errorMessage = "Only opset1 StridedSlice operation is supported";
+            return false;
+        }
+    } catch (...) {
+        return false;
+    }
+    return true;
+}
 
-void MKLDNNStridedSliceNode::getSupportedDescriptors() {
-    auto stridedSliceLayer = getCnnLayer();
+MKLDNNStridedSliceNode::MKLDNNStridedSliceNode(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache) :
+        MKLDNNNode(op, eng, cache) {
+    std::string errorMessage;
+    if (isSupportedOperation(op, errorMessage)) {
+        const auto ss = std::dynamic_pointer_cast<const ngraph::opset1::StridedSlice>(op);
+
+        auto createMask = [&](const std::vector<int64_t> &origMask, const int bit = 0, bool needReverse = false) {
+            std::vector<int> mask(origMask.begin(), origMask.end());
+            if (needReverse) {
+                for (size_t i = 0; i < mask.size(); i++)
+                    mask[i] = 1 - mask[i];
+            }
+            for (size_t i = mask.size(); i < outDims[0].ToSizeVector().size(); ++i) mask.push_back(bit);
```

Comment:
First you need to find the max rank between inDims and outDims ranks:
`const size_t nDims = std::max(inDims[DATA_ID].ndims(), outDims[0].ndims());`
As [here](https://github.com/openvinotoolkit/openvino/blob/e4aa069569d28c62ee5384c6874c4d35139952eb/inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp#L552)
And then should use `nDims` instead `outDims[0].ToSizeVector().size()` in `for`

Similarly, should replace 77th line (for `ellipsisMask`)

It should fix test on Windows

Reply:
thanks you!!!
looks like after rebase part of changes didn't applied